mkdocs.yml

```diff
@@ -8,8 +8,8 @@ nav:
     - Ibexa Developer Documentation: index.md
     - Ibexa DXP editions:
         - Editions: ibexa_products/editions.md
-        - Ibexa Headless: ibexa_products/headless.md
-        - Ibexa Experience: ibexa_products/experience.md
+        - Ibexa Headless: ibexa_products/ibexa_headless.md
+        - Ibexa Experience: ibexa_products/ibexa_experience.md
```

Comment:
When I was adding Commerce PG it turned out that 'commerce.md' file already exists.
Because of this, I decided to unify the names of all files.

Reply:
Renaming the files is fine, I don't have anything against it - I'm just wondering if we should add new redirect entries to mkdocs.yml because of that.